packages/web-components/src/components/va-back-to-top/va-back-to-top.css

```diff
@@ -0,0 +1,116 @@
+@import '../../../../../node_modules/animate.css/source/sliding_entrances/slideInUp.css';
+@import '../../../../../node_modules/animate.css/source/sliding_exits/slideOutDown.css';
```

Comment:
I added an extra `../..` at the beginning, since with the monorepo `node_modules` is at the root level.